### PR TITLE
E2e step1 review

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
- [x] Review step1
- [x] Review test template in 2 tests : CRUD-template and save-as-template

### CODE REVIEW

File: apps/delivery/delivery-e2e/src/support/app.po.ts [hunk 1]
```diff
@@ -7,7 +7,36 @@ export const getTitle = () => cy.get('section h1');
 
 export class NavbarPage {
   constructor() {
-    cy.get('header', {timeout: 60000}).contains('home');
+    cy.get('.account-icon', {timeout: 60000}).contains('account_circle');
+  }
+
+  public openLogout() {
+    cy.get('button').contains('account_circle').click();
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/app.po.ts]:
```
cy.get('.navbar button[id="account"]')
```

File: apps/delivery/delivery-e2e/src/support/app.po.ts [hunk 2]
```diff
@@ -7,7 +7,36 @@ export const getTitle = () => cy.get('section h1');
...
+  }
+
+  public clickLogout() {
+    cy.get('button').contains('Logout').click();
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/app.po.ts]: idem

File: apps/delivery/delivery-e2e/src/support/app.po.ts [hunk 3]
```diff
@@ -7,7 +7,36 @@ export const getTitle = () => cy.get('section h1');
...
+    return new LandingPage();
+  }
+
+  public clickHome() {
+    cy.get('[testId=home]').click();
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/app.po.ts]: `button[testId=home]`

File: apps/delivery/delivery-e2e/src/support/app.po.ts [hunk 4]
```diff
@@ -7,7 +7,36 @@ export const getTitle = () => cy.get('section h1');
...
+    return new HomePage();
+  }
+
+  public clickAcceptInvitationToMovie() {
+    cy.get('button').contains('Accept').last().click();
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/app.po.ts]: `.navbar .invitation button[testId="accept"]`

File: apps/delivery/delivery-e2e/src/support/app.po.ts [hunk 5]
```diff
@@ -16,15 +45,84 @@ export class NavbarPage {
...
-    return new EditProfilePage;
+    return new EditProfilePage();
+  }
+}
+
+export class TemplateDeleteModal {
+  constructor() {}
+
+  public clickConfirm() {
+    cy.get('[testId=confirm]').click();
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/app.po.ts]: button?

File: apps/delivery/delivery-e2e/src/support/app.po.ts [hunk 6]
```diff
@@ -16,15 +45,84 @@ export class NavbarPage {
...
+
+  public clickEditMaterial(value: string) {
+    cy.get('mat-card')
+    .contains(value)
+    .parent().parent()
+    .trigger('mouseover')
+    .find('button').contains('EDIT').click({force: true});
+  }
+
+  public assertMaterialChanged(value: string, description: string, category: string) {
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/app.po.ts]: assertMaterialExists?

File: apps/delivery/delivery-e2e/src/support/app.po.ts [hunk 7]
```diff
@@ -16,15 +45,84 @@ export class NavbarPage {
...
+    cy.get('h3').contains(category);
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/app.po.ts]: TODO: test that it's within a SINGLE material (you may have 3 different material with value, description, category and pass the assert).

File: apps/delivery/delivery-e2e/src/support/app.po.ts [hunk 8]
```diff
@@ -79,7 +177,7 @@ export class DeliverySettingsFormPage {
 
   public clickTeamWork() {
     cy.get('a').contains('teamwork').click();
-    return new TeamWorkPage;
+    return new DeliveryTeamWorkPage;
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/app.po.ts]: `()`

File: apps/delivery/delivery-e2e/src/support/index.ts [hunk 9]
```diff
@@ -15,3 +15,5 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands';
+
+
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/index.ts]: Nope

File: apps/delivery/delivery-e2e/src/support/app.po.ts (hunk 2 above)
Comment (Hugo-Hontans) [reply apps/delivery/delivery-e2e/src/support/app.po.ts]: Can't select the toolbar because the button is not inside

**Pull request merged**